.github/workflows/release.yml

```
@@ -0,0 +1,35 @@
+name: release
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - "*"
```

We also want to build on PRs, just not publish

```suggestion
on:
  pull_request:
  push:
    tags: ['v[0-9]+\.[0-9]+\.[0-9]+']
```